tools/alpine/packages

```diff
@@ -0,0 +1,162 @@
+alpine-baselayout
```

Comment:
I think that would be good. The current list is pretty comprehensive, but I'm sure soon we'd want to add a package or two, or maybe even remove some, so it would be good to have the full list being autogenerated